sample_code/ep244/README.md

```diff
@@ -0,0 +1,44 @@
+# [intro to terraform (intermediate)](https://youtu.be/xskzEkoisNE)
+
+Today I talk about terraform - an infrastructure-as-code tool for provisioning resources (often cloud resources)
+
+## Interactive examples
+
+### Bash
+
+```Bash
+mkdir terraform_examples
+cd !$
```

Comment:
Added command for creating a new folder for the examples -- just to prevent creating a mess in the current working directory for other users :)